azext_iot/commands.py

```diff
@@ -87,6 +87,12 @@ def load_command_table(self, _):
         cmd_group.command('send-d2c-message', 'iot_device_send_message')
         cmd_group.command('simulate', 'iot_simulate_device')
         cmd_group.command('upload-file', 'iot_device_upload_file')
+        cmd_group.command('get-parent', 'iot_device_get_parent')
```

Comment:
This is not an implementation comment. This is a requirements/design comment.

Right now I am not so sure if it makes sense for these commands to be under the `iot device` sub group when we have `iot hub device-*` and `iot edge` command groups which respectively map to device manipulation (like device identity, device twin) and edge deployment management.

These new commands are aligned to IoT Edge and as it is today `iot device` command group are operations 'as device' i.e. messaging, simulation, file upload.

See issue [#46](https://github.com/Azure/azure-iot-cli-extension/issues/46#issuecomment-425228047) for some additional details.

The core point here is the command structure needs to be organized and logically aligned. If today's structure is not intuitive then it should be improved/reformulated (more on that in the linked issue).

Reply:
@iluican and @wickste for comments.